src/amo/actions/landing.js

```diff
@@ -0,0 +1,29 @@
+import {
+  LANDING_GET,
+  LANDING_LOADED,
+  LANDING_FAILED,
+} from 'core/constants';
+
+
+export function getLanding({ addonType }) {
+  return {
+    type: LANDING_GET,
+    payload: { addonType },
+  };
+}
+
+export function loadLanding(
+  { addonType, entities, featured, highlyRated, popular }
```

Comment:
The reducer is going to throw reference errors if any one of `featured`, `highlyRated`, or `popular` is missing. If I'm correct about that, you should raise an error here if any one of those payload keys is falsey.